Dockerfile

```diff
@@ -1,14 +1,23 @@
 FROM elicocorp/odoo:7.0
 MAINTAINER Elico Corp <contact@elico-corp.com>
 
+# Set OS timezone to China
+RUN ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
+
+# Set Odoo timezone to China (will be set at startup thanks to Odoo
+# parameter substitution)
+ENV ODOO_TIMEZONE=Asia/Shanghai
```

Comment:
This is Odoo related set up and not server set up (UTC by default). This should be set by the Odoo variables only IMO.
To be discussed.